core/API/ResponseBuilder.php

```diff
@@ -164,8 +170,10 @@ private function formatExceptionMessage(Exception $exception)
 
     private function handleDataTable(DataTableInterface $datatable)
     {
-        $postProcessor = new DataTablePostProcessor($this->apiModule, $this->apiMethod, $this->request);
-        $datatable = $postProcessor->process($datatable);
+        if ($this->postProcessDataTable) {
```

Comment:
Not really. The comment would be maybe outdated at some point etc. If someone changes it for a good reason then it is ok. Just wanted to explain it why I didn't use a request var in the first place.